service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -396,10 +396,25 @@ public PipelineRun markPipelineRunSuccessAndWriteOutputs(
     pipelineOutputsRepository.save(pipelineOutput);
 
     pipelineRun.setIsSuccess(true);
```

Comment:
do we need both `isSuccess` and the `status` columns?  can we just derive if a pipeline run is successful from the status

Reply:
good call, removing isSuccess